src/sql/migrations.rs

```diff
@@ -1578,6 +1579,37 @@ fn migrate_pgp_contacts(
             .context("Step 23")?;
         let mut load_chat_contacts_stmt = transaction
             .prepare("SELECT contact_id FROM chats_contacts WHERE chat_id=? AND contact_id>9")?;
+        let is_chatmail: Option<String> = transaction
```

Comment:
Would be much simpler to pass `context.is_chatmail()` value to this function. (Config keys can also be set via env, although this is unlikely)